test/Dialect/FIRRTL/grand-central-taps.mlir

```diff
@@ -156,46 +156,14 @@ firrtl.circuit "TestHarness" attributes {
       { class = "firrtl.transforms.NoDedupAnnotation" }
     ],
     portAnnotations = [
-      [{
-      class = "sifive.enterprise.grandcentral.ReferenceDataTapKey",
-      id = 0 : i64,
-      portID = 8 : i64,
-      type = "portName" }],
-      [{
-      class = "sifive.enterprise.grandcentral.ReferenceDataTapKey",
-      id = 0 : i64,
-      portID = 7 : i64,
-      type = "portName" }],
-      [{
-      class = "sifive.enterprise.grandcentral.ReferenceDataTapKey",
-      id = 0 : i64,
-      portID = 6 : i64,
-      type = "portName" }],
-      [{
-      class = "sifive.enterprise.grandcentral.ReferenceDataTapKey",
-      id = 0 : i64,
-      portID = 5 : i64,
-      type = "portName" }],
-      [{
-      class = "sifive.enterprise.grandcentral.DataTapModuleSignalKey",
-      internalPath = "schwarzschild.no.more",
-      id = 0 : i64,
-      portID = 4 : i64 }],
-      [{
-      class = "sifive.enterprise.grandcentral.ReferenceDataTapKey",
-      id = 0 : i64,
-      portID = 3 : i64,
-      type = "portName" }],
-      [{
-      class = "sifive.enterprise.grandcentral.ReferenceDataTapKey",
-      id = 0 : i64,
-      portID = 2 : i64,
-      type = "portName" }],
-      [{
-      class = "sifive.enterprise.grandcentral.ReferenceDataTapKey",
-      id = 0 : i64,
-      portID = 1 : i64,
-      type = "portName" }]
+      [{class = "sifive.enterprise.grandcentral.ReferenceDataTapKey", id = 0 : i64, portID = 8 : i64, type = "portName"}],
+      [{class = "sifive.enterprise.grandcentral.ReferenceDataTapKey", id = 0 : i64, portID = 7 : i64, type = "portName"}],
+      [{class = "sifive.enterprise.grandcentral.ReferenceDataTapKey", id = 0 : i64, portID = 6 : i64, type = "portName"}],
+      [{class = "sifive.enterprise.grandcentral.ReferenceDataTapKey", id = 0 : i64, portID = 5 : i64, type = "portName"}],
+      [{class = "sifive.enterprise.grandcentral.DataTapModuleSignalKey", id = 0 : i64, portID = 4 : i64}],
+      [{class = "sifive.enterprise.grandcentral.ReferenceDataTapKey", id = 0 : i64, portID = 3 : i64, type = "portName"}],
+      [{class = "sifive.enterprise.grandcentral.ReferenceDataTapKey", id = 0 : i64, portID = 2 : i64, type = "portName"}],
+      [{class = "sifive.enterprise.grandcentral.ReferenceDataTapKey", id = 0 : i64, portID = 1 : i64, type = "portName"}]
```

Comment:
Nit: I wish there was some way to format this that was both terse and not requiring super long lines. I've vacillated between the format used in the change and the format used before the change. I'm fine with whatever, though. (Just a comment not a request or a suggestion for anything to change.)

Reply:
Yeah same here. Keep going back and forth. Can't decide :see_no_evil: